basic-example/Dockerfile.jupyterhub

```diff
@@ -4,9 +4,9 @@ ARG JUPYTERHUB_VERSION
 FROM jupyterhub/jupyterhub:$JUPYTERHUB_VERSION
 
 # Install dockerspawner, nativeauthenticator
+# hadolint ignore=DL3013
 RUN python3 -m pip install --no-cache-dir \
-        dockerspawner==12.* \
-        jupyterhub-nativeauthenticator==1.*
+    dockerspawner \
+    jupyterhub-nativeauthenticator
```

Comment:
We're testing latest jupyterhub version on schedule, so it should work with latest spawner and authenticator.